airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/PropertyLabel.tsx

```diff
@@ -24,7 +24,13 @@ export const PropertyLabel: React.FC<React.PropsWithChildren<PropertyLabelProps>
   children,
   htmlFor,
 }) => {
-  const examples = property._type === "formItem" || property._type === "formGroup" ? property.examples : undefined;
+  const examples =
+    // Show examples for individual items and groups, unless its a date field, since we don't
+    // want to confuse the user around potential date format examples
+    (property._type === "formItem" && property.format !== "date-time" && property.format !== "date") ||
+    property._type === "formGroup"
+      ? property.examples
+      : undefined;
```

Comment:
@timroes I don't quite get this - why would users be confused by an example date format like this?
![image](https://user-images.githubusercontent.com/7550957/203962881-e98f7315-565e-4f6a-a3bd-0a6adf7869d0.png)

Reply:
I removed this when I was still planning to do present the date inside the field in a different format. Since we're not planning this at the moment. With us sticking to present the ISO format, I agree that this can still be shown.